src/components/CustomerRating.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class CustomerRating extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      rating: '',
+      comment: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange({ target }) {
+    const { name, value } = target;
+    this.setState({
+      [name]: value,
+    });
+  }
+
+  render() {
+    const { addRatingFunction, productId } = this.props;
+    const { email, rating, comment } = this.state;
+    return (
+      <div>
+        <h1>Avaliações</h1>
+        <form>
+          <p>Intens com * são obrigatórios</p>
```

Comment:
Errinho gramatical 😉 
```suggestion
          <p>Itens com * são obrigatórios</p>
```